src/main/java/org/zalando/pazuzu/feature/FeatureDto.java

```diff
@@ -14,6 +19,16 @@
     @JsonProperty("description")
     private String description;
 
+    public List<TagDto> getTags() {
+        return (null != tags) ? tags : Collections.emptyList();
+    }
+    public void setTags(List<TagDto> tags) {
+        this.tags = tags;
+    }
+
+    @JsonProperty("tags")
+    private List<TagDto> tags;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://www.codacy.com/app/Pazuzu/pazuzu-registry/file/2750012294/issues/source?bid=3131965&fileBranchId=3355916#l30)